src/api/questionsApi.ts

```diff
@@ -1,8 +1,11 @@
 import { instance } from './Instance/instance';
-import { TestQuestionsType } from '../types/TestQuestionsType';
+import { NewQuestionType } from '../types/TestQuestionsType';
 
 export const questionsApi = {
   getQuestions() {
-    return instance.get<TestQuestionsType[]>(`questions`);
+    return instance.get<NewQuestionType[]>(`questions`);
+  },
+  getOneCard(quizId: number) {
```

Comment:
Name it getCard. It always means singular

Reply:
fixed